### ISSUE TITLE
Unit of timestamps?

### ISSUE THREAD

**Issue by erl-j** [JuanFMontesinos/Solos#3]
Hello Juan!

First and foremost I'd like to thank you for creating this excellent resource. 

I have a question concerning the timestamps in "solos_timestamps.json". 

**In what unit are the timestamps expressed?**

Thank you!

**Comment by JuanFMontesinos** [comment]
Hi, they are frame indices for each video resampled to 25 FPS. However we found some issues with the IDs and will be re-uploading some resources during this week.

**Comment by erl-j** [comment]
Ok! I found an Oboe example with id "7hZG4v0UsH4" and timestamps [[0, 2080]]. 

The end timestamp in seconds should then be 2080/25 =83.2 s. However, [the video is less than 52 seconds long](https://www.youtube.com/watch?v=7hZG4v0UsH4).  Could the id issue explain this discrepancy?

Thank you!

**Comment by JuanFMontesinos** [comment]
Yes, please feel free to check the IDs again after the next commit

**Comment by JuanFMontesinos** [comment]
Plase note that we removed the ID QL3FITw9tkM, therefore it no longer appears in either timestamps or ids json files.

BTW we further tracked the issue to a lack of resampling. Basically these videos have a different frame rate (using the proper one will lead to the correct timestamp. We will adapt it in a further commit.  They all are violin videos

```
526 23.98 PUYCwgJD2Ko
-1065 29.97 Iw91usZ7wiI
-2309 29.97 sOdmMnjIWAU
-2616 29.57 PCCkguu91r4
-2623 29.97 Qhfc41bn4jo
-1550 29.97 WlxEQT3Onlw
-3753 29.97 4emPz_bpV4E
290 23.98 a1_cmEl_ScQ
615 23.98 1ytu0LGRScg
-1046 29.97 -uV87ToKCcM
-3044 29.97 ODMV3Qrt20k
-888 29.97 Iz2SAWjD9mQ
-3324 29.97 E4GV4eqEKFQ
-1723 29.97 dTfA72RG8-g
-2345 29.97 1ndATPWcaok
-3845 29.97 a0f9W1CRxQM
-3934 29.97 3RxH4R1V_ZQ
-2863 29.97 QRdYAusDCLo
-1024 29.0 _PFXrLBWlgU
-2173 29.61 qcCPODzJNQ4
-1324 30.0 TImP-2G1FI4
-703 30.0 K0-gjCYrJn8
-3121 29.97 YRBLqsvqeMA
-25059 59.94 E1xr4touR5g
68 24.0 jsttzKQGFro
-1621 29.97 C-dhpfxFDeM
-3287 29.97 lWZG4fa3hQU
-2785 30.0 3AJYvZin3Z0
-2147 29.75 aiDmJ48o5Mw
-3224 29.97 rgmlb3jI274
-2060 29.97 JrNn2Ns7k-g
-2080 30.0 E4FEOExWpKE
-2934 30.0 LFXQZamLiw4
-1804 29.0 pGppKoQeQG8
-3991 29.97 2EVvJId3QAw
-1779 29.97 ULQIHfXmwEk
-454 29.91 MFgnRXqS6g8
-1529 30.0 BVNafswddw8
-2419 30.0 fVrk2p35zEA
-4314 29.97 zRRUSMeTwl8
-3540 29.97 XahBwMHViJg
-2404 29.97 hSPeJm9-XF4
-15242 59.94 v0Vz9bZRIQo
2051 15.0 UWVROLcT3C0
```

**Comment by JuanFMontesinos** [comment]
Soo it's finally (this time is the good one) solved at 07ca9329f9b3d35f8c37eee78cdfea6053379bc9
